Update Properties documentation with new Optimizer property (#23103)
prestodb/presto: presto-docs/src/main/sphinx/admin/properties.rst

Enable analysis and propagation of logical properties like distinct keys or cardinality among the nodes of
a query plan. The optimizer may then use these properties to perform various optimizations.

``optimizer.confidence-based-broadcasting``
^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^

* **Type:** ``boolean``
* **Default value:** ``false``

> Enable broadcasting based on the confidence of the statistics that are being used, specifically choosing to

Reviewer: ```suggestion
Enable broadcasting based on the confidence of the statistics that are being used, by
```

Author: Thanks for the comments! Making the changes now

Author: Sorry just made a small change, based on something I changed recently. should be good to go now!

> broadcast the side of a joinNode which has atleast ``HIGH`` confidence. In the case that both sides have

Reviewer: ```suggestion
broadcasting the side of a joinNode which has at least ``HIGH`` confidence. If both sides have
```
